recipes/ludoSim/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "ludoSim-jaib1" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3342abcd52a6a18e1d4b0e2e626c051f24133187647c6618c0f81d7add28d543
+
+build: 
+  number: 0
+  script: python -m pip install . --no-deps -vv 
```

Comment:
Can this be `noarch: python`.  If so, in the below you can limit python to be `- python >=3.0`

Reply:
Hi @scopatz , thanks for the feedback! When I try using `noarch: python`, I get this message from the linter:
```
noarch packages can't have selectors. If the selectors are necessary, please remove noarch: python. 
```
If my package requires python >= 3.0, how can I specify this while still using `noarch: python` ? I thought the point of `noarch: python` was to specify that this package can be run on any version of python, including 2.* ?

Reply:
Also, I'm getting [this error](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=50263&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&taskId=bef775d0-a1a2-55a3-9721-17147029baf0&lineStart=465&lineEnd=466&colStart=1&colEnd=36)

I don't know why the builder can't find my `LICENSE` file, especially because it's in my build that is on [PyPI](https://pypi.org/project/ludoSim-jaib1/). Any ideas on this?